common/screenshotter.go

```diff
@@ -121,12 +120,23 @@ func (s *screenshotter) originalViewportSize(p *Page) (*Size, *Size, error) {
 	if err != nil {
 		return nil, nil, fmt.Errorf("getting viewport dimensions: %w", err)
 	}
-	r, ok := result.(goja.Object)
+
+	r, ok := result.(map[string]interface{})
 	if !ok {
-		return nil, nil, fmt.Errorf("cannot convert to goja object: %w", err)
+		return nil, nil, fmt.Errorf("converting viewport size %q: %w", result, err)
 	}
-	viewportSize.Width = r.Get("width").ToFloat()
-	viewportSize.Height = r.Get("height").ToFloat()
+
+	width, ok := r["width"].(float64)
+	if !ok {
+		return nil, nil, fmt.Errorf("converting viewport size width %q: %w", r["width"], err)
+	}
+	viewportSize.Width = width
+
+	height, ok := r["height"].(float64)
+	if !ok {
+		return nil, nil, fmt.Errorf("converting viewport size height %q: %w", r["height"], err)
+	}
+	viewportSize.Height = height
```

Comment:
We can simply use the `convert` helper here.

Reply:
Nice, resolved in 4228c7982f6cb32eeb52d6dfcfabadfd2ffd8eb0. My concern is how it might impact the runtime to  marshal and unmarshal if we use this everywhere, but an issue for later.

Reply:
Do you mean performance-wise? But since we use Goja everywhere, that will not significantly impact us. We can only know that for sure by measuring it. Developer experience-wise, `convert` brings excellent value.